src/windows/Microsoft.Authentication.Helper.Windows/Application.cs

```diff
@@ -17,19 +19,6 @@ protected override async Task<int> RunInternalAsync(string[] args)
         {
             try
             {
-                // Listen to ADAL logs if GCM_TRACE_MSAUTH is set
-                if (Context.IsEnvironmentVariableTruthy(Constants.EnvironmentVariables.GcmTraceMsAuth, false))
```

Comment:
Are there equivalent MSAL logs that may be useful to trace?

Reply:
We are still attaching ourselves to the MSAL logger. In ADAL this was done via a static property, whereas in MSAL this must be done at 'context' construction time (see below).